src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md

```diff
@@ -379,13 +380,23 @@ Both approaches are demonstrated in the following examples of changing the templ
  </referenceBlock>
 ```
 
-In both examples, the template is specified according to the following:
+**Example 3: using the `<action method="setTemplate">` instruction**
+
+```xml
+ <referenceBlock name="page.main.title">
+   <action method="setTemplate">
+     <argument name="template" xsi:type="string">%Namespace_Module::new_template.phtml%</argument>
+   </action>
+ </referenceBlock>
+```
+
+In all listed above examples, the template is specified according to the following:
 
 -  `Namespace_Module:` defines the module the template belongs to. For example, `Magento_Catalog`.
 -  `new_template.phtml`: the path to the template relatively to the `templates` directory. It might be `<module_dir>/view/<area>/templates` or `<theme_dir>/<Namespace_Module>/templates`.
 
 {:.bs-callout-info}
-Template values specified as attributes have higher priority during layout generation, than the ones specified using `<argument>`. It means, that if for a certain block, a template is set as attribute, it will override the value you specify in `<argument>` for the same block.
+The highest priority has template that is set with setTemplate action `<action method="setTemplate">`, second priotiry has template value specified as attribute `<referenceBlock name="..." template="..."/>`, and the loweset priority has template specified using `<argument>`. It means, that if for a certain block, a template is set as attribute, it will override the value you specify in `<argument>` for the same block. In case if we have `<action method="setTemplate">` construction - it will override the values that you specified in template attribute `<referenceBlock name="..." template="..."/>` and the value you specified using `<argument>`.
```

Comment:
```suggestion
The highest priority template is one with setTemplate action `<action method="setTemplate">`, second priority has attribute specified as `<referenceBlock name="..." template="..."/>` and the lowest priority has the template using `<argument>`.

It means, that if for a certain block, a template is set as an attribute, it will override the value you specify in `<argument>` for the same block. In the case where we have `<action method="setTemplate">` construction - it will override the values that you specified in the template attribute `<referenceBlock name="..." template="..."/>` and the value you specified using `<argument>`.
```